src/components/Toast/ToastContainer.tsx

```diff
@@ -0,0 +1,24 @@
+import { useToast } from "@/hooks/useToast";
+import ToastPortal from "@/components/Toast/ToastPortal";
+import Toast from "@/components/Toast/Toast";
+
+export default function ToastContainer() {
+  const { toasts } = useToast();
+
+  return (
+    <ToastPortal>
+      <div className="fixed inset-0 z-50 flex flex-col items-center justify-center pointer-events-none">
+        {toasts.map((toast) => (
+          <div
+            key={toast.id}
+            className={`transition-opacity duration-500 ${
+              toast.isVisible ? "opacity-100" : "opacity-0"
+            }`}
+          >
+            <Toast label={toast.label} />
+          </div>
```

Comment:
`Toast` 컴포넌트에 `className` prop을 하나 추가해서 아래처럼 적용하면
불필요한 `div` 태그를 제거할 수 있을 것 같아요!

```suggestion
          <div
            key={toast.id}
          >
            <Toast 
              label={toast.label} 
              className={cn(
                "transition-opacity duration-500",
                toast.isVisible ? "opacity-100" : "opacity-0",
              )}
             />
          </div>
```

Reply:
넵! 말씀해주신 부분들 반영하여 코드 수정해보겠습니다!!😊